cluster/mesos/docker/certs/rebuild.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
Yes, copyright for new files should be the year they were penned. If it's branched or copied code, that's fine, but in general new files should match the year they were written.